modules/nomad-firewall-rules/outputs.tf

```diff
@@ -1,23 +1,23 @@
 output "firewall_rule_allow_inbound_http_url" {
-  value = "${google_compute_firewall.allow_inbound_http.self_link}"
+  value = "${google_compute_firewall.allow_inbound_http.*.self_link}"
```

Comment:
Should this be:

```hcl
 value = "${element(concat(google_compute_firewall.allow_inbound_http.*.self_link, list("")), 0)}"
```

Otherwise, we'll just return a list of 1 element, versus a string here.

Reply:
But do we want to return a string? Why should the output type change based on the number of resources that are provided?

Reply:
Hmm, I'm a little confused. The original value `"${google_compute_firewall.allow_inbound_http.self_link}"` returns a string. Your updated code `"${google_compute_firewall.allow_inbound_http.*.self_link}"` returns a list of strings.

We want to fix the splat warning, but we still want to output a string, which is why I recommended the `element(concat())` syntax above. Can you clarify what you were trying to say?

Reply:
That was sort of my point: it's more natural, in my opinion, to output a list than a string for something that may be one or more values. 
So in the case of `count=1` it should be a list of length 1, otherwise a longer list. 

Else consumers of this output would have to re-split the string when they want to use.